### DISCUSSION TITLE
Calendar time component popup - manually change time

### DISCUSSION THREAD

**Discussion by StevanZivadinovic** [primefaces/community#71] [PrimeReact] [answered]
Hi everyone,
I work with prime react calendar component and i want posability to manually change time in popup (right now i can do it in input field and when i click on arrow buttons in popup). Is there any prop which will allow me to add time via keyboard, i can't find it in documentation.
![Screenshot from 2023-09-19 15-50-07](https://github.com/primefaces/community/assets/61547542/fe47302d-51f1-4926-acbf-f0898f03a73e)

**Comment by habubey** [comment, marked as answer]
Hi there!
Unfortunately, we don't have that prop at the moment. The good news is that it will be implemented in Q4 along with Accessibility updates this year.

**Reply by StevanZivadinovic** [reply]
Thank you very much. If you can inform me here when the prop becomes available.

**Reply by habubey** [reply]
I'll inform indeed 👍🏼

**Reply by yoni-edrigo** [reply]
@habubey I will be happy to be informed too if it`s possible for you 🙏
